extensions/html-language-features/package.json

```diff
@@ -188,6 +188,12 @@
           "default": true,
           "description": "%html.autoClosingTags%"
         },
+        "html.doNotAddAttributeQuotes": {
```

Comment:
this should follow the style of the existing settings, e.g. html.completion.attributeQuotes